package.json

```diff
@@ -1,21 +1,21 @@
 {
   "dependencies": {
-    "@azure-tools/typespec-autorest": "^0.48.0",
-    "@azure-tools/typespec-azure-core": "^0.48.0",
-    "@azure-tools/typespec-azure-resource-manager": "^0.48.0",
-    "@azure-tools/typespec-client-generator-core": "^0.48.4",
-    "@typespec/compiler": "^0.62.0",
-    "@typespec/http": "^0.62.0",
-    "@typespec/http-client-csharp": "^0.1.9-alpha.20241211.3",
-    "@typespec/http-server-csharp": "^0.58.0-alpha.5",
-    "@typespec/http-server-javascript": "file:./pkg/typespec-http-server-javascript-0.58.0-alpha.5.tgz",
-    "@typespec/http-client-java": "^0.1.3",
-    "@azure-tools/typespec-ts": "^0.35.0",
-    "@typespec/http-client-python": "^0.4.2",
-    "@typespec/json-schema": "^0.62.0",
-    "@typespec/openapi": "^0.62.0",
-    "@typespec/openapi3": "^0.62.0",
-    "@typespec/rest": "^0.62.0",
-    "@typespec/versioning": "^0.62.0"
+    "@azure-tools/typespec-autorest": "^0.49.0",
+    "@azure-tools/typespec-azure-core": "^0.49.0",
+    "@azure-tools/typespec-azure-resource-manager": "^0.49.0",
+    "@azure-tools/typespec-client-generator-core": "^0.49.0",
+    "@typespec/compiler": "^0.63.0",
+    "@typespec/http": "^0.63.0",
+    "@typespec/http-client-csharp": "^0.1.9-alpha.20241216.3",
+    "@typespec/http-server-csharp": "^0.58.0-alpha.6",
+    "@typespec/http-server-javascript": "^0.58.0-alpha.6",
```

Comment:
Was this released?  I didn't find it in npmjs.

Reply:
https://www.npmjs.com/package/@typespec/http-server-javascript